parsons/google/google_sheets.py

```diff
@@ -395,6 +399,148 @@ def format_cells(self, spreadsheet_id, range, cell_format, worksheet=0):
         ws.format(range, cell_format)
         logger.info("Formatted worksheet")
 
+    def attempt_gsheet_method(self, method, i=1, max=6, wait_time=15, **kwargs):
```

Comment:
Can we adjust this to also take an `*args` splat variable? That's the standard way to solve the positional arguments problem that you note in the docstring. The Python standard for function passthrough like you're doing here is to do:

```python
def call_foo(my_function, *args, **kwargs):
    # do stuff...
    my_function(*args, **kwargs)
```